dpe/src/dpe_instance.rs

```diff
@@ -23,6 +23,7 @@ pub struct DpeInstance<'a, C: Crypto> {
     contexts: [Context; MAX_HANDLES],
     support: Support,
     localities: &'a [u32],
+    has_initialized: bool,
```

Comment:
Might be good to have a comment to explain this (something about how `InitializeContext(simulation=false)` is only allowed once)

Reply:
Thats a good idea. I added a couple comments.